src/test/resources/CsvParserTest/require_trailing_whitespaces/repoconfig_redundantLines_test.csv

```diff
@@ -0,0 +1,6 @@
+  
+
+
+                   
+Repository's Location,Branch,File formats,Ignore Glob List,Ignore standalone config,Ignore Commits List
+https://github.com/reposense/testrepo-Beta.git,master,,,,
```

Comment:
please add several more lines with spaces in between as well

Reply:
I've added 2 more repositories and several blank / empty lines.